tests/PermissionRegistarTest.php

```diff
@@ -48,4 +52,43 @@ public function it_can_check_uids()
             $this->assertFalse(PermissionRegistrar::isUid($not_uid));
         }
     }
+
+    /** @test */
+    public function it_can_get_permission_class() {
+        $this->assertSame(SpatiePermission::class, app(PermissionRegistrar::class)->getPermissionClass());
+        $this->assertSame(SpatiePermission::class, get_class(app(PermissionContract::class)));
+    }
+
+    /** @test */
+    public function it_can_change_permission_class() {
```

Comment:
Should these 2 "change" tests "set" it to something else first, and test that both changes produce the expected results?

Reply:
I had forgotten about these tests, I added them to avoid breaking the functionality, but they have default values, after "set", it changes to the new ones, but i gonna add previous expected values